app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsSummary.kt

```diff
@@ -0,0 +1,51 @@
+package org.wikipedia.suggestededits
+
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.restbase.page.RbPageSummary
+import org.wikipedia.gallery.ExtMetadata
+import org.wikipedia.gallery.ImageInfo
+import org.wikipedia.page.Namespace
+import org.wikipedia.page.PageTitle
+import org.wikipedia.util.StringUtil
+
+class SuggestedEditsSummary {
+
```

Comment:
This could use the `data class` syntax where all the member fields are declared on one line.

Reply:
Sure.